BMS Workspace-git/bms-1227-fw/Phantom_drivers/drivers/include/Current_transducer.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Current_transducer.h
```

Comment:
Rename it  to "current_transducer.h". Same thing for the .c file.

Reply:
- renamed both files to "current_transducer.h" and "current_transducer.c" respectively